### PR TITLE
KevS-SK1739-Creation-Associate-Domain-with-Google-Site-for-DomainDNS-TRANSLATIONS

### PR DESCRIPTION
**Pull request by Kazarus4**
All translations are done.

Need proof by : 

- @tcpdumpfbacke for DE sub
- @Y0Coss or @BrenKt for latin and PL subs

### CODE REVIEW

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.es-es.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+> [!warning]
+>
+> - El servicio de asistencia Google Site no tiene acceso a los parámetros de su dominio de OVHcloud y, por lo tanto, no puede aconsejarle sobre la información que deba proporcionarle.
+>
+> - La configuración, la gestión y la responsabilidad de los servicios que OVHcloud pone a su disposición recaen sobre usted. Por lo tanto, usted deberá asegurarse de que estos funcionen correctamente.<br><br> Esta guía le ayudará a realizar las operaciones más habituales. No obstante, si necesita ayuda, le recomendamos que contacte con un [proveedor especializado](/links/partner) o con el editor del servicio. Nosotros no podremos asistirle al respecto. Para más información, consulte la sección [Más](#gofurther) información de esta guía.
+>
+
+## Requisitos
+
+- Haber iniciado sesión en el [área de cliente de OVHcloud](/links/manager){.external}.
```
Comment (Y0Coss) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.es-es.md:26]: We don't need the `{.external}` tags anymore, especially on links towards our own domains.

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.es-es.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+Si tiene un dominio con OVHcloud y quiere conectarlo a un Google Site, Esta guía explica los pasos necesarios para preparar y configurar la zona DNS de OVHcloud con el fin de permitir la configuración de un Google Site.
+
+**Cómo conectar un dominio de OVHcloud a un Google Site**
+
+> [!warning]
+>
+> - El servicio de asistencia Google Site no tiene acceso a los parámetros de su dominio de OVHcloud y, por lo tanto, no puede aconsejarle sobre la información que deba proporcionarle.
+>
+> - La configuración, la gestión y la responsabilidad de los servicios que OVHcloud pone a su disposición recaen sobre usted. Por lo tanto, usted deberá asegurarse de que estos funcionen correctamente.<br><br> Esta guía le ayudará a realizar las operaciones más habituales. No obstante, si necesita ayuda, le recomendamos que contacte con un [proveedor especializado](/links/partner) o con el editor del servicio. Nosotros no podremos asistirle al respecto. Para más información, consulte la sección [Más](#gofurther) información de esta guía.
```
Comment (Y0Coss) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.es-es.md]:
```suggestion
> - La configuración, la gestión y la responsabilidad de los servicios que OVHcloud pone a su disposición recaen sobre usted. Por lo tanto, usted deberá asegurarse de que estos funcionen correctamente.<br><br> Esta guía le ayudará a realizar las operaciones más habituales. No obstante, si necesita ayuda, le recomendamos que contacte con un [proveedor especializado](/links/partner) o con el editor del servicio. Nosotros no podremos asistirle al respecto. Para más información, consulte la sección [Más información](#gofurther)  de esta guía.
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Comment connecter un nom de domaine OVHcloud à un Google Site"
 excerpt: "Préparez et configurez la zone DNS de votre nom de domaine OVHcloud pour le connecter à un Google Site"
-updated: 2024-09-26
+updated: 2024-10-03
```
Comment (Y0Coss) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md]: Why changing the date? 
If no new information is added, the date should be kept as 2024-09-26 (same thing for all translations)

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.it-it.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+Se il dominio è registrato in OVHcloud e vuoi connetterlo a un Google Site. Questa guida ti mostra la procedura da seguire per preparare e configurare la zona DNS di OVHcloud e configurare il tuo Google Site.
+
+**Come connettere un dominio OVHcloud a un Google Site**
+
+> [!warning]
+>
+> - L’assistenza Google Site non ha accesso ai parametri del dominio OVHcloud e non può quindi consigliarti quali informazioni è necessario fornire.
+>
+> - OVHcloud mette a tua disposizione servizi di cui tu sei responsabile per la configurazione e la gestione. Garantirne quotidianamente il corretto funzionamento è quindi responsabilità dell’utente.<br><br> Questa guida ti aiuta a realizzare le operazioni più ricorrenti. Tuttavia, in caso di difficoltà o dubbi, ti consigliamo di contattare un [fornitore specializzato](/links/partner) o il fornitore del servizio. OVHcloud non potrà fornirti alcuna assistenza. Per maggiori informazioni consulta la sezione [Per saperne](#gofurther) di più.
```
Comment (Y0Coss) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.it-it.md]:
```suggestion
> - OVHcloud mette a tua disposizione servizi di cui tu sei responsabile per la configurazione e la gestione. Garantirne quotidianamente il corretto funzionamento è quindi responsabilità dell’utente.<br><br> Questa guida ti aiuta a realizzare le operazioni più ricorrenti. Tuttavia, in caso di difficoltà o dubbi, ti consigliamo di contattare un [fornitore specializzato](/links/partner) o il fornitore del servizio. OVHcloud non potrà fornirti alcuna assistenza. Per maggiori informazioni consulta la sezione [Per saperne di più](#gofurther).
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.pl-pl.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+Posiadasz domenę w OVHcloud i chcesz ją połączyć z Google Site. W tym przewodniku znajdziesz etapy przygotowania i konfiguracji strefy DNS OVHcloud, aby umożliwić konfigurację Google Site.
+
+**Dowiedz się, jak powiązać domenę OVHcloud z Google Site**
+
+> [!warning]
+>
+> - Pomoc Google Site nie ma dostępu do ustawień Twojej domeny OVHcloud, a zatem nie może udzielić Ci porad dotyczących informacji, które należy jej dostarczyć.
+>
+> - OVHcloud udostępnia różnorodne usługi, jednak to Ty odpowiadasz za ich konfigurację i zarządzanie nimi. Ponosisz więc odpowiedzialność za ich prawidłowe funkcjonowanie.<br><br> Oddajemy w Twoje ręce niniejszy przewodnik, którego celem jest pomoc w wykonywaniu bieżących zadań. Niemniej jednak, w przypadku trudności zalecamy skontaktowanie się z wyspecjalizowanym [usługodawcą](/links/partner) lub skontaktuj się z dostawcą usługi. Niestety firma OVH nie będzie mogła udzielić wsparcia w tym zakresie. Więcej informacji znajduje się w sekcji [Sprawdź](#gofurther) również ten przewodnik.
```
Comment (Y0Coss) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.pl-pl.md]:
```suggestion
> - OVHcloud udostępnia różnorodne usługi, jednak to Ty odpowiadasz za ich konfigurację i zarządzanie nimi. Ponosisz więc odpowiedzialność za ich prawidłowe funkcjonowanie.<br><br> Oddajemy w Twoje ręce niniejszy przewodnik, którego celem jest pomoc w wykonywaniu bieżących zadań. Niemniej jednak, w przypadku trudności zalecamy skontaktowanie się z wyspecjalizowanym [usługodawcą](/links/partner) lub skontaktuj się z dostawcą usługi. Niestety firma OVH nie będzie mogła udzielić wsparcia w tym zakresie. Więcej informacji znajduje się w sekcji [Sprawdź również](#gofurther) ten przewodnik.
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.pl-pl.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+
+## Sprawdź również <a name="go-further"></a>
+
+[Zmiana serwerów DNS domeny OVHcloud](/pages/web_cloud/domains/dns_server_general_information)
+
+[Utworzenie strefy DNS OVHcloud](/pages/web_cloud/domains/dns_zone_create)
+
+[Edycja strefy DNS OVHcloud](/pages/web_cloud/domains/dns_zone_edit)
+
+Aby zmienić sposób zarządzania domeną na inne konto klienta OVHcloud, zapoznaj się z przewodnikiem "[Zarządzanie kontaktami dla usług](/pages/account_and_service_management/account_information/managing_contacts) OVHcloud".
```
Comment (Y0Coss) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.pl-pl.md]:
```suggestion
Aby zmienić sposób zarządzania domeną na inne konto klienta OVHcloud, zapoznaj się z przewodnikiem "[Zarządzanie kontaktami dla usług OVHcloud](/pages/account_and_service_management/account_information/managing_contacts)".
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.pt-pt.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+
+Se possui um nome de domínio na OVHcloud, deseja ligá-lo a um Google Site. Neste guia, encontrará as etapas de preparação e de configuração da sua zona DNS da OVHcloud para permitir a configuração do seu Google Site.
+
+**Saiba como ligar o seu domínio OVHcloud a um Google Site**
+
+> [!warning]
+>
+> - O suporte Google Site não tem acesso aos parâmetros do seu nome de domínio OVHcloud e não pode, por isso, aconselhá-lo sobre as informações que deverá fornecer.
+>
+> - A responsabilidade sobre a configuração e a gestão dos serviços que a OVHcloud disponibiliza recai sobre o utilizador. Assim, deverá certificar-se de que estes funcionam corretamente.<br><br> Este guia fornece as instruções necessárias para realizar as operações mais habituais. No entanto, se encontrar dificuldades, recomendamos que recorra a um [prestador de serviços especializado](/links/partner) e/ou que contacte o editor do serviço. Não poderemos proporcionar-lhe assistência técnica. Para mais informações, consulte a secção [Quer saber mais](#gofurther)? deste guia.
```
Comment (Y0Coss) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.pt-pt.md]:
```suggestion
> - A responsabilidade sobre a configuração e a gestão dos serviços que a OVHcloud disponibiliza recai sobre o utilizador. Assim, deverá certificar-se de que estes funcionam corretamente.<br><br> Este guia fornece as instruções necessárias para realizar as operações mais habituais. No entanto, se encontrar dificuldades, recomendamos que recorra a um [prestador de serviços especializado](/links/partner) e/ou que contacte o editor do serviço. Não poderemos proporcionar-lhe assistência técnica. Para mais informações, consulte a secção [Quer saber mais?](#gofurther) deste guia.
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.pt-pt.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+> **Etapa 4**
+>> **Registo CNAME**<br><br>
+>> Para identificar os registos "CNAME" existentes, clique no menu Filtros na parte superior da tabela de registos DNS e selecione "CNAME`".
+>>
+>> ![dnszone](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-zone/filter-cname.png){.thumbnail}
+>>
+>> - Clique no botão `...`{.action} à direita da linha da tabela correspondente ao seu subdomínio em "www" (exemplo: `www.mydomain.ovh.`) e depois clique em `Modificar entrada`{.action}.<br>
+>> - Se não tiver um registo "CNAME" existente, clique no botão `Adicionar uma entrada`{.action} no canto superior direito do ecrã e selecione o "Campo de apontamento" CNAME`{.action}.
+>>
+>> Preencha o campo **Subdomínio** com o valor `www` e introduza `ghs.googlehosted.com.` no campo **Destino**. Uma vez que estes valores têm de ser alterados, verifique-os na página "[Valores dos registos CNAME](https://support.google.com/a/answer/112038?sjid=3052810298579211755-EU)" da documentação oficial<br>
```
Comment (Y0Coss) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.pt-pt.md]:
```suggestion
>> Preencha o campo **Subdomínio** com o valor `www` e introduza `ghs.googlehosted.com.` no campo **Destino**. Uma vez que estes valores têm de ser alterados, verifique-os na página "[Valores dos registos CNAME](https://support.google.com/a/answer/112038?sjid=3052810298579211755-EU)" da documentação oficial.<br>
```

File: pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.pt-pt.md [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+>> - É necessário criar um registo do tipo "TXT". Clique no botão `Adicionar uma entrada`{.action} no canto superior direito do ecrã e selecione o "Campo" apontador `TXT`{.action}.
+>> Preencha os campos **Subdomínio** e **Destino** com as informações presentes na página "[Valores dos registos TXT](https://support.google.com/a/answer/2716802?hl=pt&ref_topic=2716886&sjid=3052810298579211755-EU)" da documentação oficial. Normalmente, o valor do campo **Subdomínio** está vazio, e o campo **Destino** é do tipo `google-site-verification=XXXXXXXXXXXX`.<br>
+>> Clique em `Seguinte`{.action} para validar o seu registo "TXT" e passe à etapa 4.
+>>
+> **Etapa 4**
+>> **Registo CNAME**<br><br>
+>> Para identificar os registos "CNAME" existentes, clique no menu Filtros na parte superior da tabela de registos DNS e selecione "CNAME`".
```
Comment (Y0Coss) [inline pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.pt-pt.md]:
```suggestion
>> Para identificar os registos "CNAME" existentes, clique no menu Filtros na parte superior da tabela de registos DNS e selecione "CNAME".
```

**Comment by Kazarus4** [comment]
for latin and PL subs, corrections done. It still proof for DE and EN subs before merge.

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**